pynitrokey/cli/fido2.py

```diff
@@ -181,7 +181,7 @@ def list():
 
 @click.command()
 @click.option("--count", default=8, help="How many bytes to generate (defaults to 8)")
-@click.option("-s", "--serial", help="Serial number of Nitrokey to use")
+@click.option("-s", "--serial", help="Serial number of Nitrokey to use. Prefix with 'device=' to provide device file, e.g. 'device=/dev/hidraw5'.. Prefix with 'device=' to provide device file, e.g. 'device=/dev/hidraw5'.")
```

Comment:
Is the duplication in the help text intended?

Reply:
That was my first idea, however I wanted to avoid adding new switch to every possible command, and pass the new variable deep to device opening function. So piggy-backing the `serial` switch is more conservative in the introduced changes, and that's the advantage. I would go with separate switch, if that would be specified only once, e.g. with:
- https://click.palletsprojects.com/en/8.0.x/advanced/#forwarding-unknown-options

I did not want to spend too much time on changing the implementation because of this though.

Alternatively with the current design, perhaps it would possible to prepare `common` switches, and then decorate with them every subcommand. Still that would add at least 2 more modified lines per command.